third_party/wff/specification/validator/src/main/java/com/samsung/watchface/DWFValidationApplication.java

```diff
@@ -77,8 +82,7 @@ private static void printOptions() {
 
     private static void printUsage() {
         System.out.println("Usage : " +
-                "java -jar dwf-format-" + MAX_SUPPORTED_FORMAT_VERSION +
-                "-validator-" + APPLICATION_VERSION + ".jar" +
+                "java -jar " + APPLICATION_NAME + ".jar " +
                 " <format-version> <any options> <your-watchface.xml> <more-watchface.xml> ...");
```

Comment:
Should we update readme as well?

https://github.com/google/watchface/tree/19e7d427500c1a95ae66e51f397e7d2694744440/third_party/wff

Reply:
Apologies,updated